examples/nlp/text_normalization_as_tagging/helpers.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+from typing import Tuple
+
+import pytorch_lightning as pl
+from omegaconf import DictConfig
+
+from nemo.collections.nlp.models import ThutmoseTaggerModel
+from nemo.utils import logging
+
+__all__ = ['ITN_MODEL', 'MODEL_NAMES', 'instantiate_model_and_trainer']
+
+ITN_MODEL = 'itn'
+MODEL_NAMES = [ITN_MODEL]
+
+
+def instantiate_model_and_trainer(
```

Comment:
this is used only in train.py, could you move it there and delete this file?

Reply:
No, it is used in train.py and infer.py